src/util.h

```diff
@@ -7,13 +7,25 @@
 #include <sstream>
 #include <string>
 #include <vector>
+#include <stdio.h>
+#include <fcntl.h>
+#include <sys/mman.h>
+#include "light_elf.h"
 
 namespace appimage {
     namespace update {
-        static void removeNewlineCharacters(std::string& str) {
-            str.erase(std::remove(str.begin(), str.end(), '\n'), str.end());
-        }
+        static std::vector<std::string> split(const std::string& s, char delim = ' ') {
```

Comment:
You moved this function up for some reason I don't understand. Please move it to its original position.